dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Api::V1::SectionsController < ApplicationController
   layout false
 
+  # Don't bother redirecting to login when denying access to the JSON APIs
+  rescue_from CanCan::AccessDenied do
+    head :forbidden
+  end
```

Comment:
Cool!  I wonder if this is worth putting in an `BaseApiController` (along with `layout false`).  Then having controllers in this folder inherit from that.

(Future cleanup, shouldn't block this PR.)